manifests/puppet.pp

```diff
@@ -43,11 +43,11 @@
     pubkey { $ssl_ca_cert:
       key_pair => $::certs::server_ca,
     } ~>
-    file { $client_key:
-      ensure => file,
-      owner  => '52',
-      mode   => '0400',
+    file { [$client_cert, $client_key, $ssl_ca_cert]:
+      ensure  => file,
+      owner   => 'puppet',
+      mode    => '0400',
+      require => Class['puppet::server'],
```

Comment:
Using @ehelms original idea from https://github.com/Katello/puppet-certs/pull/113, we can't rely on the UID until https://tickets.puppetlabs.com/browse/SERVER-1381 is fixed.